charts/move2kube/values.schema.json

```diff
@@ -0,0 +1,184 @@
+                    "title": "The move2kubeURL Schema",
+                    "examples": [
+                        "https://move2kube-route-sonataflow-infra.apps.cluster-8xfw.redhatworkshops.io"
+                    ]
+                },
+                "backstageNotificationURL": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The backstageNotificationURL Schema",
+                    "examples": [
+                        "http://orchestrator-backstage.orchestrator/api/notifications/"
+                    ]
+                }
+            },
+            "examples": [{
+                "name": "serverless-workflow-m2k",
+                "image": "quay.io/orchestrator/serverless-workflow-move2kube:latest",
+                "brokerURL": "http://broker-ingress.knative-eventing.svc.cluster.local/sonataflow-infra/default",
+                "move2kubeURL": "https://move2kube-route-sonataflow-infra.apps.cluster-8xfw.redhatworkshops.io",
+                "backstageNotificationURL": "http://orchestrator-backstage.orchestrator/api/notifications/"
+            }]
+        },
+        "kfunction": {
+            "type": "object",
+            "default": {},
+            "title": "The kfunction Schema",
+            "required": [
+                "name",
+                "image"
+            ],
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The name Schema",
+                    "examples": [
+                        "m2k-save-transformation-func"
+                    ]
+                },
+                "image": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The image Schema",
+                    "examples": [
+                        "quay.io/orchestrator/serverless-workflow-m2k-kfunc:latest"
+                    ]
+                }
+            },
+            "examples": [{
+                "name": "m2k-save-transformation-func",
+                "image": "quay.io/orchestrator/serverless-workflow-m2k-kfunc:latest"
+            }]
+        },
+        "instance": {
+            "type": "object",
+            "default": {},
+            "title": "The instance Schema",
+            "required": [
+                "name",
+                "image"
+            ],
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The name Schema",
+                    "examples": [
+                        "move2kube"
+                    ]
+                },
+                "image": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The image Schema",
+                    "examples": [
+                        "quay.io/orchestrator/move2kube-ui:latest"
```

Comment:
do we want to put our image in example or the konveyor one? At some point I hope my PR will be merged